### PR TITLE
fix: comment downtime sync test and restore helm ac tests

### PR DESCRIPTION
**Pull request by xpuska513**
## What
* Restore helm AC test functionality in workflow
* Disable testDowntimeDuringSync test due to it being flaky

### CODE REVIEW

**Review by c-p-b — APPROVED** [review]

**Review by supertopher — APPROVED** [review]
looks great.  thanks for making this change Kyryl

File: airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/ContainerOrchestratorAcceptanceTests.java [hunk 1]
```diff
@@ -100,44 +100,45 @@ static void end() {
...
-    final long numAttempts = apiClient.getJobsApi()
-        .getJobInfo(new JobIdRequestBody().id(connectionSyncRead.getJob().getId()))
-        .getAttempts()
-        .size();
-
-    // it should be able to accomplish the resume without an additional attempt!
-    assertEquals(1, numAttempts);
-  }
+  // This test is flaky.  It is commented out until that condition us understood
+  // See: https://github.com/airbytehq/airbyte-cloud/issues/3618
```
Comment (supertopher) [inline airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/ContainerOrchestratorAcceptanceTests.java:104]:
```suggestion
  // See: https://github.com/airbytehq/airbyte/issues/19948
```

**Review by supertopher — COMMENTED** [review]
changed link to more specific story

I think making issues for specific broken tests helps us to keep from losing track of these.  we need to ideally tell the author of the failing test as well

File: airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/ContainerOrchestratorAcceptanceTests.java (hunk 1 above)
Comment (davinchia) [inline airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/ContainerOrchestratorAcceptanceTests.java:104]: can we use the @Disabled annotation instead?

Comment (xpuska513) [reply airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/ContainerOrchestratorAcceptanceTests.java:104]: 👌

**Comment by supertopher** [comment]
There is alot of movement on this from master @xpuska513 what are we waiting on for this change?

File: airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/ContainerOrchestratorAcceptanceTests.java [hunk 2]
```diff
@@ -100,9 +101,10 @@ static void end() {
   void setup() throws URISyntaxException, IOException, SQLException {
     testHarness.setup();
   }
-
+  // This test is flaky.  It is commented out until that condition us understood
+  // See: https://github.com/airbytehq/airbyte/issues/19948
   @Test
-  @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+  @Disabled
```
Comment (supertopher) [inline airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/ContainerOrchestratorAcceptanceTests.java:107]:
```suggestion
  @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
```

**Review by supertopher — APPROVED** [review]
plx look at my comment before merging

**Pull request merged**